src/triage/experiments/multicore.py

```diff
@@ -2,15 +2,25 @@
 import traceback
 from functools import partial
 from pebble import ProcessPool
+from multiprocessing.reduction import ForkingPickler
 
 from triage.component.catwalk.utils import Batch
 
 from triage.experiments import ExperimentBase
 
 
 class MultiCoreExperiment(ExperimentBase):
-    def __init__(self, n_processes=1, n_db_processes=1, *args, **kwargs):
-        super(MultiCoreExperiment, self).__init__(*args, **kwargs)
+    def __init__(self, db_engine, n_processes=1, n_db_processes=1, *args, **kwargs):
+        try:
+            ForkingPickler.dumps(db_engine)
+        except Exception as e:
+            raise ValueError(
+                "multiprocessing is unable to pickle passed SQLAlchemy engine."
+                "use triage.create_engine instead when running MultiCoreExperiment: "
+                "(e.g. from triage import create_engine)"
+            ) from e
+
+        super(MultiCoreExperiment, self).__init__(db_engine=db_engine, *args, **kwargs)
```

Comment:
and of course then here:

```suggestion
        super(MultiCoreExperiment, self).__init__(config, db_engine, *args, **kwargs)
```